app.js

```diff
@@ -162,3 +162,9 @@ TeamValidator.PM.spawn();
  *********************************************************/
 
 require('./repl').start('app', cmd => eval(cmd));
+
+/*********************************************************
+ * Load the GitHub plugin
+ *********************************************************/
+
+Object.assign(Chat.commands, require('./chat-plugins/github').commands);
```

Comment:
apparently, calling `require('chat-plugins/github')` during a chat hotpatch causes a crash. so, we call it only once here